queries.sql

```diff
@@ -43,3 +43,11 @@ SELECT neutered, COUNT(*) AS escape_count  FROM animals WHERE escape_attempts >
  SELECT species , MIN(weight_kg) AS min_weight , MAX(weight_kg) AS  max_weight FROM animals GROUP BY species;
  SELECT species , MIN(weight_kg) AS min_weight , MAX(weight_kg) AS  max_weight FROM animals GROUP BY species;
  SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id WHERE s.name = 'Pokemon';
+SELECT o.full_name , a.name AS animal_name FROM owners o JOIN animals a ON a.owner_id = o.id;
```

Comment:
**OPTIONAL** - You have done well so far, but from the screenshot attached i your PR, the answer to this query is wrong, you are to get all owners and their animals including owners without an animal. Your answer is missing `Jodie Whittaker' who does not own an animal. Please fix this 🙏